<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(88, 88, 88);
</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="Title_Bar">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>20</number>
       </property>
       <property name="leftMargin">
        <number>20</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Montserrat</family>
           <pointsize>15</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255,255,255);
background: transparent;</string>
         </property>
         <property name="text">
          <string>ID phòng: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="ID_Room">
         <property name="font">
          <font>
           <family>Montserrat</family>
           <pointsize>15</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255,255,255);</string>
         </property>
         <property name="text">
          <string>[CODE]</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="btn_wrap">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>10</number>
          </property>
          <item>
           <widget class="QPushButton" name="btn_minimize">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	background: transparent;
	border-radius: 15px;
}
QPushButton:hover {
	background: rgb(108, 108, 108);
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="Resources.qrc">
              <normaloff>:/images/icons/minimize_white.png</normaloff>:/images/icons/minimize_white.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_quit">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	background: transparent;
	border-radius: 15px;
}
QPushButton:hover {
	background: rgb(255, 0, 0);
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="Resources.qrc">
              <normaloff>:/images/icons/close_white.png</normaloff>:/images/icons/close_white.png</iconset>
            </property>
            <property name="shortcut">
             <string>Alt+X</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="Main_Frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="1">
        <widget class="QFrame" name="student_list_frame">
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-radius: 8px;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="student_label">
            <property name="minimumSize">
             <size>
              <width>240</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Montserrat</family>
              <pointsize>15</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255,255,255);</string>
            </property>
            <property name="text">
             <string>Danh sách học sinh</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="student_list">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>iCiel Effra Regular</family>
              <pointsize>15</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QListWidget {
	padding: 10px;
	background-color: rgb(255, 255, 255);
}

QListWidget::item {
	border-radius: 5px;
	margin: 0px 0px 10px 0px;	
	background-color: rgb(193, 193, 193);
}

QLineEdit {
	color: rgb(0,0,0);
	background-color: transparent;
}

QLabel {
	color: rgb(0,0,0);
	background-color: transparent;
}

QScrollBar{
	background : rgb(255, 255, 255);	
}

QScrollBar::handle{
	background : rgb(158, 158, 158);
	border-radius: 5px;
	margin: 3px 3px 3px 3px;
}

QScrollBar::handle::pressed{
	background : rgb(85, 85, 85);
}

QScrollBar::add-line {
	height: 0px;
}

QScrollBar::sub-line {
	height: 0px;
}

QScrollBar::add-page, QScrollBar::sub-page {
	height: 0px;
}</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="kick_btn">
            <property name="minimumSize">
             <size>
              <width>204</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>iCiel Effra Regular</family>
              <pointsize>15</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	background-color: rgb(255, 255, 255);
	border-radius: 10px;
	color: rgb(0,0,0);
}

QPushButton:hover {
	color: rgb(255, 255, 255);
	background-color: rgba(0, 0, 0, 150);
}

QPushButton:pressed {
	color: rgb(255, 255, 255);
	background-color: rgb(0, 0, 0);
}r</string>
            </property>
            <property name="locale">
             <locale language="Vietnamese" country="Vietnam"/>
            </property>
            <property name="text">
             <string>Kick học sinh</string>
            </property>
            <property name="iconSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="reload_btn">
            <property name="minimumSize">
             <size>
              <width>204</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>iCiel Effra Regular</family>
              <pointsize>15</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	background-color: rgb(255, 255, 255);
	border-radius: 10px;
	color: rgb(0,0,0);
}

QPushButton:hover {
	color: rgb(255, 255, 255);
	background-color: rgba(0, 0, 0, 150);
}

QPushButton:pressed {
	color: rgb(255, 255, 255);
	background-color: rgb(0, 0, 0);
}r</string>
            </property>
            <property name="locale">
             <locale language="Vietnamese" country="Vietnam"/>
            </property>
            <property name="text">
             <string>Load lại danh sách</string>
            </property>
            <property name="icon">
             <iconset resource="Resources.qrc">
              <normaloff>:/images/icons/loading.png</normaloff>:/images/icons/loading.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QFrame" name="lesson_list_frame">
         <property name="styleSheet">
          <string notr="true">border-radius: 8px;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="2" column="0">
           <widget class="QPushButton" name="add_btn">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	background-color: rgb(255, 255, 255);
	border-radius: 10px;
	color: rgb(0,0,0);
}

QPushButton:hover {
	color: rgb(255, 255, 255);
	background-color: rgba(0, 0, 0, 150);
}

QPushButton:pressed {
	color: rgb(255, 255, 255);
	background-color: rgb(0, 0, 0);
}r</string>
            </property>
            <property name="locale">
             <locale language="Vietnamese" country="Vietnam"/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="Resources.qrc">
              <normaloff>:/images/icons/plus.png</normaloff>:/images/icons/plus.png</iconset>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QListWidget" name="lesson_list">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>iCiel Effra Regular</family>
              <pointsize>15</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QListWidget {
	padding: 10px;
	background-color: rgb(255, 255, 255);
}

QListWidget::item {
	border-radius: 5px;
	margin: 0px 0px 10px 0px;	
	background-color: rgb(193, 193, 193);
}

QLineEdit {
	color: rgb(0,0,0);
	background-color: transparent;
}

QLabel {
	color: rgb(0,0,0);
	background-color: transparent;
}

QScrollBar{
	background : rgb(255, 255, 255);	
}

QScrollBar::handle{
	background : rgb(158, 158, 158);
	border-radius: 5px;
	margin: 3px 3px 3px 3px;
}

QScrollBar::handle::pressed{
	background : rgb(85, 85, 85);
}

QScrollBar::add-line {
	height: 0px;
}

QScrollBar::sub-line {
	height: 0px;
}

QScrollBar::add-page, QScrollBar::sub-page {
	height: 0px;
}</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QPushButton" name="download_btn">
            <property name="minimumSize">
             <size>
              <width>130</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>iCiel Effra Regular</family>
              <pointsize>15</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	background-color: rgb(255, 255, 255);
	border-radius: 10px;
	color: rgb(0,0,0);
}

QPushButton:hover {
	color: rgb(255, 255, 255);
	background-color: rgba(0, 0, 0, 150);
}

QPushButton:pressed {
	color: rgb(255, 255, 255);
	background-color: rgb(0, 0, 0);
}r</string>
            </property>
            <property name="locale">
             <locale language="Vietnamese" country="Vietnam"/>
            </property>
            <property name="text">
             <string>Tải xuống bài học</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="lessong_label">
            <property name="minimumSize">
             <size>
              <width>240</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Montserrat</family>
              <pointsize>15</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255,255,255);</string>
            </property>
            <property name="text">
             <string>Danh sách bài học</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QPushButton" name="del_lesson_btn">
            <property name="minimumSize">
             <size>
              <width>130</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>iCiel Effra Regular</family>
              <pointsize>15</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	background-color: rgb(255, 255, 255);
	border-radius: 10px;
	color: rgb(0,0,0);
}

QPushButton:hover {
	color: rgb(255, 255, 255);
	background-color: rgba(0, 0, 0, 150);
}

QPushButton:pressed {
	color: rgb(255, 255, 255);
	background-color: rgb(0, 0, 0);
}r</string>
            </property>
            <property name="locale">
             <locale language="Vietnamese" country="Vietnam"/>
            </property>
            <property name="text">
             <string>Xóa bài học</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QFrame" name="lesson_list_frame_2">
         <property name="minimumSize">
          <size>
           <width>210</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>210</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-radius: 8px;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="1" column="0">
           <widget class="QListWidget" name="Achievements_list">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>iCiel Effra Regular</family>
              <pointsize>15</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QListWidget {
	padding: 10px;
	background-color: rgb(255, 255, 255);
}

QListWidget::item {
	border-radius: 5px;
	margin: 0px 0px 10px 0px;	
	background-color: rgb(193, 193, 193);
}

QLineEdit {
	color: rgb(0,0,0);
	background-color: transparent;
}

QLabel {
	color: rgb(0,0,0);
	background-color: transparent;
}

QScrollBar{
	background : rgb(255, 255, 255);	
}

QScrollBar::handle{
	background : rgb(158, 158, 158);
	border-radius: 5px;
	margin: 3px 3px 3px 3px;
}

QScrollBar::handle::pressed{
	background : rgb(85, 85, 85);
}

QScrollBar::add-line {
	height: 0px;
}

QScrollBar::sub-line {
	height: 0px;
}

QScrollBar::add-page, QScrollBar::sub-page {
	height: 0px;
}</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="Achievements">
            <property name="minimumSize">
             <size>
              <width>240</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Montserrat</family>
              <pointsize>15</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255,255,255);</string>
            </property>
            <property name="text">
             <string>Xếp hạng</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="ButtonFrame">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 8px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="4">
        <widget class="QPushButton" name="download_info_btn">
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>iCiel Effra Regular</family>
           <pointsize>15</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	background-color: rgb(255, 255, 255);
	border-radius: 10px;
	color: rgb(0,0,0);
}

QPushButton:hover {
	color: rgb(255, 255, 255);
	background-color: rgba(0, 0, 0, 150);
}

QPushButton:pressed {
	color: rgb(255, 255, 255);
	background-color: rgb(0, 0, 0);
}r</string>
         </property>
         <property name="locale">
          <locale language="Vietnamese" country="Vietnam"/>
         </property>
         <property name="text">
          <string>Tải xuống dữ liệu bài làm học sinh</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
